teralaunch/src/utils.js

```
@@ -0,0 +1,127 @@
+  }
+  return `${size.toFixed(2)} ${units[unitIndex]}`;
+}
+
+/**
+ * Formats a given number of bytes per second into a human-readable speed string.
+ *
+ * @param {number} bytesPerSecond the number of bytes per second to format
+ * @returns {string} the formatted speed string
+ */
+export function formatSpeed(bytesPerSecond) {
+  if (!isFinite(bytesPerSecond) || bytesPerSecond < 0) return "0 B/s";
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)